index.html

```diff
@@ -1518,28 +1512,18 @@ <h4>CoAP Vocabulary Terms</h4>
             </tr>
             <tr>
               <td><code>cov:options</code></td>
-              <td>CoAP options sent with the message, e.g., <code>[ { "cov:optionNumber": 6, "cov:optionValue": 0 }]</code> to observe.</td>
+              <td>CoAP options sent with the message, e.g., <code>[ { "cov:optionName": "Observe", "cov:optionValue": 0 }]</code> to observe.</td>
```

Comment:
Since Observe is more suitable to be done with `"subprotocol":"cov:observe"`, should this example better use a different CoAP option?

Reply:
I agree :) What options would be useful and give-able in such a small example? I have thought of  `If-Match` or `Max-Age`

Reply:
`[ { "cov:optionName": "Accept", "cov:optionValue": 110 }]` maybe? (Max-Age is not valid in requests. Don’t know if If-Match is a good or bad example.)

One interesting question here is: If the consumer does not understand what `”Accept”` means and is just supposed to include this option with this name in its request (which is legitimate in case of `Accept`), how does it find out that the option number is 17 and the option value format is `uint`?

Reply:
I find your proposal better ^^ and put it in https://github.com/w3c/wot-binding-templates/pull/82/commits/e8517b122888c30bffe30b1b7324539cd0d049f5

Reply:
Regarding the question, I would suggest that the client should have a mapping of these names to the actual protocol header values. I have looked at [node-coap](https://github.com/mcollina/node-coap#setOption) implementation where one would actually use the names in the API and the value also as a string instead of the `uint`. I think that similar to the `methodName`, the implementations just use the more human-readable way to specify things.

Reply:
But wasn't the scenario for `cov:options` that the client is unaware of that CoAP option? If the client is aware, why not simply `"cov:ocf-content-format": 12345` instead of `"cov:options": [ { "cov:optionName": "OCF-Content-Format", "cov:optionValue": 12345 }]`? For the method, the client always has to be aware of what it means, so that's different.

Reply:
I think this is a question of whether the client is aware that `ocf-content-format` is an option or not. I am not sure why this array notation was picked in the beginning. Probably to give the hint that "these are options, go figure out how to put them into your request". 

I think that having them inside the array looks more contained but bulkier. In the end, it doesn't help the fact that the client has to know how the options map to its client implementation. I still think that a more human readable way is better since some libraries that build the requests with a function that gets the options in the `optionName` kind of way. Golang implementation and the node.js implementation do this for example. Whereas some libraries will do the `optionNumber` kind of way. Californium is one example, see [here](https://github.com/eclipse/californium/blob/master/californium-core/src/main/java/org/eclipse/californium/core/CoapClient.java#L498). So I think it would be better to have `name` instead of `number` since we can somewhat assume that a user of a library with `number` would have more knowledge of the protocols.